docs/INSTALL.md

```diff
@@ -20,6 +20,8 @@ is actually newer in version number, than what was there already.
 
 ### Installation on Linux
 
+**WARNING: This manual process is more difficult**; the `install.sh` script is recommended, please go back to the [README](../README.md) unless you're sure you need to do this.
```

Comment:
Nit: This warning is valid for the Mac OS guide too right? Should we put it before `### Installation on Linux` so that it references both?
Or we can repeat it below in the Mac section.

Reply:
I've done this in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/781/commits/60b31e276bb4929855e6bcc4b4a27b076c1386f1 but just a sanity check: the install via install.sh does actually work on macOS, right? I know it is intended to.